programmers/난이도별/level01.시저_암호/JiaLee0707.js

```diff
@@ -0,0 +1,11 @@
+function solution(s, n) {
+    return s.split('').map((r) => {
+        if (r === ' ') return r;
+
+        let upperCaseASCII  = (r.toUpperCase()).charCodeAt();
+        if (upperCaseASCII + n > 'Z'.charCodeAt()) r = r.charCodeAt() + n - 26;
```

Comment:
```suggestion
        const alphabetCount = 26;
        const upperCaseASCII  = (r.toUpperCase()).charCodeAt();
        
        if (upperCaseASCII + n > 'Z'.charCodeAt()) r = r.charCodeAt() + n - alphabetCount;
```
`upperCaseASCII`의 경우 선언된 뒤 재할당 할 필요가 없기에 `const`를 사용하는 것이 [의도치 않은 재할당을 방지해주기 때문에 더 안전해보여요.](https://velog.io/@bathingape/JavaScript-var-let-const-%EC%B0%A8%EC%9D%B4%EC%A0%90) 개인적인 의견이지만 let과 const를 적절하게 사용하면 이해하기도 더 편하더라구요!

추가로 [매직넘버의 경우 상수로 선언하여 사용](https://javabom.tistory.com/28)하는 것이 좋아요. 지금은 그럴 일이 없겠지만 유지보수가 필요할 경우 다른 사람이 `26`이라는 숫자를 보고 어떤 수인지 확신할 수 없기 때문이에요.
이번 `26`이라는 숫자는 제 예상으로 알파벳 개수로 판단되어 위와 같이 수정해보았어요.

Reply:
지금까지 변수 선언을 할 때, 기본으로 `let`을 사용했는데
앞으로는 적절하게 사용하도록 노력해야겠네요!
코드 리뷰 감사합니다!!